src/components/ProtectedContent/ProtectedComponent.tsx

```diff
@@ -4,17 +4,24 @@ import { PROTECTED_TYPE, ProtectedContent } from './ProtectedContent';
 
 interface IProtectedComponentProps {
   role?: AUTH_ROLE;
+
+  // hide component, otherwise disabled component is visible
   hide?: boolean;
-  disable?: boolean;
 }
 
+/**
+ * Protected component content is automatically disabled when some of the requirements (for example role) are not fulfilled.
```

Comment:
This is not correct. Some components should not be disabled, such as link buttons, which automatically redirect to Keycloak login page.